test.txt

```diff
@@ -0,0 +1,9 @@
+Nakon šest dana struče prakse više nisam imao čiste odječe te sam slijedeći put ponio više kompleta kako bi se što bolje obuko.
```

Comment:
Please add this to a samples directory with the dictionary.
See: [da_DK - Danish](https://github.com/streetsidesoftware/cspell-dicts/tree/master/dictionaries/da_DK)